pkg/asset/agent/image/unconfigured_ignition.go

```diff
@@ -185,24 +185,26 @@ func (a *UnconfiguredIgnition) Generate(_ context.Context, dependencies asset.Pa
 		// Enable the agent-check-config-image.service for the current workflow.
 		enabledServices = append(enabledServices, "agent-check-config-image.service")
 
-	case workflow.AgentWorkflowTypeInstallInteractiveDisconnected:
-		// Add the rendezvous host file. Agent TUI will interact with that file in case
-		// the rendezvous IP wasn't previously configured, by managing it as a template file.
-		if rendezvousIP == "" {
-			rendezvousHostFile := ignition.FileFromString(rendezvousHostEnvPath, "root", 0644, rendezvousHostTemplateData)
-			config.Storage.Files = append(config.Storage.Files, rendezvousHostFile)
+		// To Do:
+		if sentinelfiel is present or another way to identify we want to invoke AgentWorkflowTypeInstallInteractiveDisconnected workflow {
```

Comment:
There's no way to tell if a sentinel file is present, because it can only be added to the ignition after we generate the ignition here.
I don't think we need it though.